data/brands/shop/curtain.json

```diff
@@ -16,6 +16,16 @@
         "name": "Heytens",
         "shop": "curtain"
       }
-    }
+    },
+    {
+      "displayName": "Vianney",
+      "locationSet": {"include": ["mx"]},
+      "tags": {
+        "brand": "Vianney",
+        "brand:wikidata": "Q113203804",
+        "name": "Vianney",
+        "shop": "fabric"
```

Comment:
This will be changed to shop=curtain when run formatter.

Reply:
Fixed